### PR TITLE
Add Kubernetes 1.12.5

### PR DESCRIPTION
**Pull request by superseb**
https://github.com/rancher/rancher/issues/15489

### CODE REVIEW

File: infra-templates/k8s/56/docker-compose.yml.tpl [hunk 1]
```diff
@@ -66,8 +66,8 @@ kubelet:
         - /run:/run:rprivate
         - /var/run:/var/run:rprivate
         - /sys:/sys:ro,rprivate
-        - /var/lib/docker:/var/lib/docker:rprivate
-        - /var/lib/kubelet:/var/lib/kubelet:shared
+        - /var/lib/docker:/var/lib/docker:rw,rslave,z
```
Comment (alena1108) [inline infra-templates/k8s/56/docker-compose.yml.tpl:69]: could you explain the need for this change?

**Comment by alena1108** [comment]
@superseb could you update the pr to k8s 1.12.5 as it's been released already

File: infra-templates/k8s/56/docker-compose.yml.tpl (hunk 1 above)
Comment (superseb) [reply infra-templates/k8s/56/docker-compose.yml.tpl:69]: This breaks on newer Docker versions as daemon root, I changed it to be in line with RKE.

**Comment by alena1108** [comment]
LGTM

**Pull request merged**